### DISCUSSION TITLE
support for fontWeight: "inherit"

### DISCUSSION THREAD

**Discussion by monicakogler** [outline/rich-markdown-editor#220] [General] [answered]
Version 9 supported passing a string in the theme's fontWeight so that it could inherit the style from the rest of the app. Is there a simple way to do the same in version 10?

Thanks!

**Comment by tommoor** [comment, marked as answer]
It _should_ still be supported

https://github.com/outline/rich-markdown-editor/blob/49fb0180b11c930dbb374a868a811587f2e17d62/src/index.tsx#L574

**Comment by monicakogler** [comment]
Ah, yeah I think it's an issue with the types:

```
    Types of property 'fontWeight' are incompatible.
      Type 'string' is not assignable to type 'number'.
```

**Reply by tommoor** [reply]
Interesting, there is no explicit type for the theme so I guess it's inferring…
